WordPress/Classes/ViewRelated/Post/Preview/PreviewWebKitViewController.swift

```diff
@@ -287,4 +290,8 @@ extension PreviewWebKitViewController {
         setWidth(selectedDevice.width)
         webView.evaluateJavaScript(selectedDevice.viewportScript, completionHandler: nil)
     }
+
+    func webView(_ webView: WKWebView, didFinish navigation: WKNavigation!) {
+        webView.alpha = 1
```

Comment:
I've tested this out, but I'm still experiencing a flicker for some reason (especially noticeable when switching between `Tablet` and `Desktop` preview modes).

I thought that perhaps this `webView.apha = 1` should be in the `completionHandler` for the JavaScript injection, but that did not help either. Then I tried commenting out the above line, as a "sanity check", expecting the view to be blank, and that surprisingly had no effect :thinking: . I also tried explicitly setting `webView.isOpaque = false` in `viewDidLoad()`, but that _also_ had no effect. At this point, I'm a bit confused why alpha seems to not be respected :man_shrugging: 

If we can't find a reliable way to get alpha working to smooth out the jitter on the Swift side of things, perhaps we could find a work-around with the script injection code? I also wonder if it is worth revisiting the need for the injected script, but that's probably a bit out of scope for this issue.

Tanner, I wonder, do you observe a difference when you comment out the above line? (i.e. `// webView.alpha = 1`)?

Reply:
Thanks @mkevins for giving it a look! That's odd.

> Then I tried commenting out the above line, as a "sanity check", expecting the view to be blank, and that surprisingly had no effect 🤔

I get a blank page as expected when commenting out `webView.alpha = 1`.

You probably saw this, but just in case: This fix currently doesn't apply to previewing a site on site creation, that's going to be addressed in [a separate issue](https://github.com/wordpress-mobile/WordPress-iOS/issues/17864). Sorry for any confusion there. This PR is only for the areas outside of that:

- Tapping the URL at the top of the "My Site" view.
- Tapping the "View Site" row at the bottom of the "My Site" view.
- Tapping the three dots at the top right and choosing "Preview" while editing a Post or Page.
- (any other non-site creation areas I may have missed)

If we're on the same page and this issue is still occurring, would you mind sharing:
- the theme you were previewing
- the device size and orientation

Thanks! 🙇

Reply:
I've combined the two issues in to this PR, so now it targets both template and site previews.

Reply:
> This fix currently doesn't apply to previewing a theme on site creation,

:man_facepalming: that explains it! I was trying via the Site Creation flow. Will try again with these latest updates. No wonder commenting it out had no effect :laughing: .